lib/src/token.dart

```diff
@@ -127,26 +127,21 @@ class ScalarToken implements Token {
 enum TokenType {
   streamStart,
   streamEnd,
-
```

Comment:
I'm guessing the blank lines split the types into groups that made sense to someone. Consider restoring them.

Reply:
Done!  (I wanted to blame formatting but looks like I did it after all? 🤷 )